src/store/slices/CommonSilce.ts

```diff
@@ -1,9 +1,16 @@
 import { PayloadAction, createSlice } from "@reduxjs/toolkit";
```

Comment:
the file should start with the small letter

Reply:
done